tests/php/Authenticator/LoginHandlerTest.php

```diff
@@ -43,16 +43,6 @@ protected function setUp(): void
 
         EnforcementManager::config()->set('enabled', true);
 
-        Injector::inst()->load([
-            Security::class => [
-                'properties' => [
-                    'authenticators' => [
-                        'default' => '%$' . MemberAuthenticator::class,
```

Comment:
This configuration isn't needed anymore, because sudo mode is activated by the session login handler which is enabled by default.